docs/gathered-data.md

```diff
@@ -314,9 +314,6 @@ None
 ### Notes
 PersistentVolumes was included at `4.7.0` and backported to `4.6.20+`.
 
-**Conditional data**: If the Image Registry configuration uses any PersistentVolumeClaim for the storage,
-the corresponding PersistentVolume definition is gathered
```

Comment:
I still think this is important information about the gatherer so I don't know why we are removing?

Reply:
Thanks